docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
While pre-produced responses can be great for performance and availability it also brings a challenge for CA's that have issued many millions or even billions of certificates (IoT) that are infrequently accessed. Pre-produced fresh responses can take much more resources than a solid cache layer.

In general pre-production of responses will cause less fresh response information and bigger spikes to CDN resources for status information that is used by many clients as all clients have the same next update (they will randomize but in a much shorter timeframe).

Can we instead state that the responder should ignore nonce when provided by the client? While this will still impact the cache on most CDN's, the nonce can be filtered by a smart cache layer for both pre-produced and normal responses before sending it to any live responder.

If pre-produced, I think we should also state that a status should be reproduced and published when the status of a certificate is changed.